site/content/integrate/apps/quickstart/quick-start-python/_index.md

```diff
@@ -66,11 +66,11 @@ The App has to provide a manifest, which declares the App's metadata required fo
 
 ### Bindings and locations
 
-This App adds a channel header button and slash commands. In order to register these locations, there is a `POST` handler for the `/bindings` endpoint in your App's API.
+[Bindings]({{< ref "/integrate/apps/structure/bindings" >}}) specify how an App's calls should be displayed and invoked from these locations. This App adds a channel header button and slash commands. In order to register these locations, there is a `POST` handler for the `/bindings` endpoint in your App's API.
```

Comment:
What locations are being referred to by the phrase "these locations"?
It doesn't look like locations were introduced before this instance.
Could we add some text about what locations are?